### PR TITLE
Statistics added

### PR DESCRIPTION
**Pull request by Liana2707**

### CODE REVIEW

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt [hunk 1]
```diff
@@ -32,6 +35,98 @@ class PDFDocument(val name: String = "PDF",
...
+		for (section in pdfsections)
+			if (section.title[2].isDigit())
+				continue
+			else {
+				newpdfStructure.add(section)
+			}
+		for (i in 0 until newpdfStructure.size) {
+			var sectionSizeInPage: Int
+			var sectionSizeInPercents: Float
+			val documentSize = pdfSize
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt:114]: tumblr_8bc50ff60f567ed7ad3bd29e9eaa75e0_de48e27f_400

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt [hunk 2]
```diff
@@ -31,4 +31,10 @@ class Checker {
 			it.process(document)
 		}.flatten().toSet().toList()
 	}
+	fun getPDFStatistic(pdfName: String) : Statistic {
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/Checker.kt:34]: Checker class should check PDF for rule violations

Suggestion: Add StatisticsService or a similar class that handles acquiring statistics

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/statistics/SectionStatistics.kt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.github.darderion.mundaneassignmentpolice.checker.statistics
+import com.github.darderion.mundaneassignmentpolice.pdfdocument.text.Section
+
+data class SectionStatistics(var section: Section, var sectionSizeInPage: Int,
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/statistics/SectionStatistics.kt:4]: "sectionSizeInPercents" can be calculated based on two other properties

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/statistics/SectionStatistics.kt [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+                             var sectionSizeInPercents: Float, var documentSize:Int)
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/checker/statistics/SectionStatistics.kt:5]: "Val" should be used by default as it makes code a lot more clear about which properties can change and which fields stay the same after the construction of an object

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt [hunk 5]
```diff
@@ -32,6 +35,98 @@ class PDFDocument(val name: String = "PDF",
...
+							allWords[word] = 1
+						}
+					}
+				}
+			}
+		val topKWords = allWords.toList().sortedByDescending { (k, v) -> v }.toMap()
+		return WordsStatistic(wordCount,topKWords)
+	}
+	fun getPageStatistic(): PageStatistic {
+		val pdfsections = PDFStructure(text).sections
```
Comment (Darderion) [inline src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt:101]: camelCase

**Review by Darderion — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt [hunk 6]
```diff
@@ -32,6 +35,98 @@ class PDFDocument(val name: String = "PDF",
 	fun print() {
 		text.map { "${it.area} | ${it.text.joinToString("--") { "${it.font.size}-${it.text}"}}" }.forEach(::println)
 	}
+	fun getWordsStatistic() : WordsStatistic
```
[the remark this answers is not in the document]
Comment (Liana2707) [reply src/main/kotlin/com/github/darderion/mundaneassignmentpolice/pdfdocument/PDFDocument.kt]: Done

**Pull request merged**